modules/cnv-define-tcp-liveness-probe.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-monitoring-vm-health.adoc
+
+[id="cnv-define-tcp-liveness-probe_{context}"]
+
+= Define a TCP liveness probe
+
+This procedure provides an example configuration file for defining
+TCP liveness probes.
+
+.Procedure
+
+. Customize a YAML configuration file to create an TCP liveness probe, using
+this code block as an example. In this example:
+* You configure a probe using `spec.livenessProbe.tcpSocket`, which queries port `1500` of the
+`VirtualMachineInstance`, after an initial delay of `120` seconds.
+* The `VirtualMachineInstance` installs and runs a minimal HTTP server
+on port `1500` using `cloud-init`.
++
+----
+apiVersion: kubevirt.io/v1alpha3
+kind: VirtualMachineInstance
+metadata:
+  labels:
+    special: vmi-fedora
+  name: vmi-fedora
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+    resources:
+      requests:
+        memory: 1024M
+  livenessProbe:
+    initialDelaySeconds: 120
+    periodSeconds: 20
+    tcpSocket:
+      port: 1500
+    timeoutSeconds: 10
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - name: containerdisk
+    registryDisk:
+      image: kubevirt/fedora-cloud-registry-disk-demo
+  - cloudInitNoCloud:
+      userData: |-
+        #cloud-config
+        password: fedora
+        chpasswd: { expire: False }
+        bootcmd:
+          - setenforce 0
+          - dnf install -y nmap-ncat
+          - systemd-run --unit=httpserver nc -klp 1500 -e '/usr/bin/echo -e HTTP/1.1 200 OK\\n\\nHello World!'
+    name: cloudinitdisk
+----
++
+. Create the `VirtualMachineInstance` by running the following command:
++
+----
+$ oc create -f
```

Comment:
This should be something like:
`$ oc create -f <file name>.yaml`

Reply:
Done.